IABot/checkIfDead.php

```diff
@@ -132,6 +132,12 @@ protected function processResult( $headers, $curlerrno, $url ) {
 			} else {
 				return true;
 			}
+			// Check for error messages in redirected URL string
+		} elseif ( strpos( $effectiveUrlClean, '404.htm' ) !== false ||
+				   strpos( $effectiveUrlClean, '/404/' ) !== false ||
+				   stripos( $effectiveUrlClean, 'error' ) !== false ||
```

Comment:
Wouldn't this line mark URLs like http://expressjs.com/en/guide/error-handling.html as dead?